src/test/java/com/google/api/codegen/protoannotations/testdata/library_v2_gapic.yaml

```diff
@@ -0,0 +1,693 @@
+type: com.google.api.codegen.ConfigProto
+config_schema_version: 2.0.0
+language_settings:
+  java:
+    package_name: com.google.example.library.v1
+    interface_names:
+      google.example.library.v1.LibraryService: Library
+    release_level: GA
+  python:
+    package_name: google.cloud.example.library_v1.gapic
+  go:
+    # Intentionally changed from the gRPC package name.
+    # See: https://github.com/googleapis/toolkit/issues/320
+    package_name: cloud.google.com/go/library/apiv1
+    domain_layer_location: cloud.google.com/go/library
+    interface_names:
+      google.example.library.v1.LibraryService: LibClient
+  ruby:
+    # Note that package_name for ruby is intentionally changed
+    # from the gRPC package name of the API to test the generated
+    # code. See: https://github.com/googleapis/toolkit/issues/309
+    package_name: Library::V1
+  # Use default PHP package name.
+  nodejs:
+    package_name: library.v1
+    domain_layer_location: google-cloud
+  csharp:
+    package_name: Google.Example.Library.V1
+interfaces:
+  - name: google.example.library.v1.MyProto
+    retry_codes_def:
```

Comment:
Is this necessary? We should be able to omit the retry codes from v2 config if they just specify the defaults.

Reply:
Done.